React/Base/RCTUtils.h

```diff
@@ -62,4 +62,7 @@ RCT_EXTERN NSError *RCTErrorWithMessage(NSString *message);
 RCT_EXTERN id RCTNullIfNil(id value);
 RCT_EXTERN id RCTNilIfNull(id value);
 
+// Create NSURL for jsCodeLocation in AppDelegate:application
+RCT_EXTERN NSURL *RCTWebSocketExecutorURL(NSString *appendString);
```

Comment:
Let's rename this to something less WebSocketExecutor-specific. Maybe `RCTPackagerURL`?

small nit: the argument can actually be named "path", because that's what it is.